packages/communication-react/review/communication-react.api.md

```diff
@@ -952,7 +931,7 @@ export type MessageContentType = 'text' | 'html' | 'richtext/html' | 'unknown';
 // @public
 export type MessageProps = {
     message: ChatMessage | SystemMessage | CustomMessage;
-    strings: ILocaleKeys;
+    dateStrings: MessageThreadStrings;
```

Comment:
One of the conclusions from the meeting was to standardize on a `props.strings` field with the relevant fields here.